src/containers/exchange/index.js

```diff
@@ -70,15 +69,15 @@ class ExchangeHome extends React.Component {
                                     orders={orders}
                                     rates={rates}
                                     userAccountAddress={userAccount.address}
-                                    header="All orders"
+                                    header="Order book"
                                 />
                             </Pgrid.Column>
                         </Pgrid.Row>
                         <Pgrid.Row>
                             <TradeHistory
                                 trades={trades}
                                 userAccountAddress={userAccount.address}
-                                header="Trades history"
+                                header="My trade history"
```

Comment:
in general this should be `Your ...` everywhere. For consistency leave it `My trade history` for now and I'll create a ticket.